File: assignment 10 - C# practice problems/18.cs
//Write a c# function to get the number of days in a month. 

using System;
namespace MyNameSpace
{
    class Program18
    {
        public static void NumberOfDaysInAMontnDriverFunction()
        {

            Console.WriteLine("Enter year");
            var yearInput = Console.ReadLine();
            Console.WriteLine("Enter month");
            var monthInput = Console.ReadLine();

            var daysInMonth = DateTime.DaysInMonth(int.Parse(yearInput), int.Parse(monthInput));

            Console.WriteLine("Cuurent date is " + daysInMonth);


        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/18.js
//Write a JavaScript function to get the number of days in a monthInput. 

function numberOfDaysInAMontnDriverFunction() {
    function numberOfDaysInAMonth(monthInput, year) {
        let dateInput = new Date(year, monthInput, 0);
        return dateInput.getDate();
    }

    let yearInput = prompt("Enter year");
    let monthInput = prompt("Enter month");

    alert(numberOfDaysInAMonth(monthInput, yearInput));
}

numberOfDaysInAMontnDriverFunction();